cc_library/js-scc/index.js

```diff
@@ -0,0 +1,325 @@
+const Paho = require("paho-mqtt");
+
+/**
+ * Message is a class containing all info required by the message_manual.md
+ */
+class Message {
+  constructor(device_id, type, contents) {
+    this.device_id = device_id;
+    this.time_sent = formatDate(new Date());
+    this.type = type;
+    this.contents = contents;
+  }
+}
+
+/**
+ * Abstract device class from which all custom devices should inherit.
+ * Defines all required methods needer for communication to S.C.I.L.E.R.
+ * @Abstract
+ */
+class Device {
```

Comment:
this should be a separate class

Reply:
it is a separate class :P
I shall put this class in a different file

Reply:
This costs me no too much time. 
I made an issue for this GH-151